src/executables/util_service.c

```diff
@@ -1362,20 +1327,14 @@ process_service (int command_type, bool process_window_service)
 	{
 	  if (!are_all_services_stopped (0, process_window_service))
 	    {
+	      (void) process_javasp (command_type, 0, NULL, false, process_window_service);
```

Comment:
I think that javasp will stop a process server() or process heartbeat().
Is this code for stopping the javasp if db server doesn't execute and javasp only execute ?

Reply:
If a DB server name is not registered in the service section, DB server is not actually stopped by `cubrid service stop` command. Because the master server shuts down by the command, the server loses connection with the master and halts.
Therefore, this line is needed to explicitly pass the stop command.

Reply:
OK, i got.